RELEASES.md

```diff
@@ -1,3 +1,82 @@
+Version 1.31.0 (2018-12-06)
+==========================
+
+Language
+--------
+- [This version marks the release of the 2018 edition of Rust.][54057]
+- [There is now an explicit attribute for declaring a trait is a
+  _"marker"_ trait.][53693] Using `#[marker]` will declare that the trait is a
+  marker trait which requires that the trait doesn't have any items and that
+  impls must be empty. This allows you to have overlapping implementations.
+- [New lifetime elision rules now allow for eliding lifetimes in functions and
+  impl headers.][54778] E.g. `impl<'a> Reader for BufReader<'a> {}` can now be
+  `impl Reader for BufReader {}`. Lifetimes are still required to be defined
+  in structs.
+- [You can now define and use constant functions.][54835] These are currently
```

Comment:
I feel this should be a bigger deal even if limited (because limitation removal is being worked on...) so I'd move this point upwards (it should be the 2nd item). :)

Reply:
There's never really been a order of importance, it has usually been PR number in ascending order. It doesn't matter to me either way.